rules-tattletale/addon/src/main/java/org/jboss/windup/rules/apps/tattletale/TattletaleRuleProvider.java

```diff
@@ -58,8 +60,13 @@ public Configuration getConfiguration(RuleLoaderContext ruleLoaderContext)
         @Override
         public void perform(GraphRewrite event, EvaluationContext context)
         {
-            Boolean generateReport = (Boolean) event.getGraphContext().getOptionMap().get(EnableTattletaleReportOption.NAME);
-            if (generateReport == null || !generateReport)
+            Boolean enableReport = (Boolean) event.getGraphContext().getOptionMap().get(EnableTattletaleReportOption.NAME);
+            Boolean disableReport = (Boolean) event.getGraphContext().getOptionMap().get(DisableTattletaleReportOption.NAME);
+            Collection<String> targets = (Collection<String>) event.getGraphContext().getOptionMap().get(TargetOption.NAME);
+            boolean eapTarget = targets.stream().anyMatch(target -> target.startsWith("eap"));
+            if (eapTarget && disableReport != null && disableReport && (enableReport == null || !enableReport))
```

Comment:
for a tiny bit more readability I would use Boolean.valueOf(disableReport) here and for the enableReport logic as well